doc/scripts/run-all.sh

```diff
@@ -8,10 +8,12 @@ ulimit -s 524288 # For 2-sat test
 for test in $tests; do
     echo "$(basename $test): "
     start=`date +%s.%N`
-    g++ -std=c++14 -O2 $test && ./a.out
+    g++ -std=c++14 -O2 $test
+    retCode1=$?
+    ./a.out
```

Comment:
it doesn't make sense to run this if the compile failed. I thought &? would do the right thing with &&?

Reply:
I was doing stupid stuff before.